src/main.rs

```diff
@@ -4,5 +4,5 @@ async fn run() -> Result<(), ()> {
 
 #[tokio::main]
 async fn main() {
-    let _result = run().await;
+    run().await.unwrap();
```

Comment:
Why to unwrap? to propagate errors to the top?

Reply:
Well, since we don't do anything else at the moment, I'd simply `unwrap` for the time being.